src/fastjet/__init__.py

```diff
@@ -41,7 +41,28 @@
 from fastjet._swig import GridMedianBackgroundEstimator  # noqa: F401, E402
 from fastjet._swig import IndexedSortHelper  # noqa: F401, E402
 from fastjet._swig import InternalError  # noqa: F401, E402
-from fastjet._swig import JetDefinition  # noqa: F401, E402
+from fastjet._swig import JetDefinition as JetDefinitionNoCast  # noqa: F401, E402
+
+
+class JetDefinition(JetDefinitionNoCast):
```

Comment:
Is there any reason to have this up here? or can we move it after all of the imports are done?